compose/docker-compose.base.yml

```
@@ -11,8 +11,30 @@ networks:
+
+  metabase-data:
+    driver: local
+    name: metabase
+
+  redis-data:
+    driver: local
+    name: redis-data
+
+  redis-insight:
+    driver: local
+    name: redis-insight
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)